aptos-move/e2e-move-tests/Cargo.toml

```diff
@@ -34,6 +34,7 @@ claims = { workspace = true }
 hex = { workspace = true }
 itertools = { workspace = true }
 move-binary-format = { workspace = true }
+move-command-line-common = { workspace = true }
```

Comment:
We really should stop using this. I know that @brmataptos recommended you to use the env var reader there, but this code will go away at some point. Good to submit but in the future lets avoid it.